### PR TITLE
Project structure

### PR DESCRIPTION
**Pull request by sntgopl**
In this milestone I have:

 - Created HTML and CSS basic structure.
 - Added leaderboard list storage.
 - Created a class for leaders with add and clear methods.
 - Create a function for HTML writing.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @sntgopl Kindly add a descriptive readme to your pr, this is required for a review

**Comment by prolajumokeoni** [comment]
Hi @sntgopl  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive readme, kindly use the [microverse readme](https://github.com/microverseinc/readme-template) template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.html [hunk 1]
```diff
@@ -3,33 +3,28 @@
...
+					</div>
+			</div>
+			<div class="container-right">
+					<h2>Add your score</h2>
+					<input type="text" id="name" placeholder="Name">
+					<input type="text" id="score" placeholder="Score">
+					<button id="addbtn">Add score</button>
+			</div>
+	</div>
 </body>
```
Comment (BenMukebo) [inline src/index.html]: - [OPTIONAL] Be careful with blank lines and indentation. Do not add blank lines, spaces, or indentations without a reason. For readability, add blank lines to separate large or logical code blocks and add two spaces of indentation. Do not use the tab key 👍🏻

File: src/index.html [hunk 2]
```diff
@@ -3,33 +3,28 @@
...
+							<ul id="leaderboard">
+							</ul>
+					</div>
+			</div>
+			<div class="container-right">
+					<h2>Add your score</h2>
+					<input type="text" id="name" placeholder="Name">
+					<input type="text" id="score" placeholder="Score">
+					<button id="addbtn">Add score</button>
+			</div>
```
Comment (BenMukebo) [inline src/index.html:31]: - [ ] Great job so far 👏🏻 However, According to the project requirements, you should write plain HTML markup with minimum styling. Kindly make sure your HTML code looks like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), styling is not required for this project


image

- Make sure to add ramdom player's name and their scrore as shown on the originl template:

Screenshot from 2022-10-13 04-28-30

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @sntgopl,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- You have setup html, css and JavaScript linters correctly. ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (sntgopl) [reply src/index.html]: I'm not sure why in Github it appears like that @BenMukebo. If you clone the repo it doesn't, I can't change it from local.

**Review by roshan-bajgain — COMMENTED** [review]
Hi @sntgopl, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter fails. please run this `npx stylelint "**/*.{css,scss}"` on your terminal and fix the stylelint problem and submit for re-review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Reem-lab — APPROVED** [review]
# Hi @sntgopl,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌 Professional README file ✅


## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**